spring-cloud-azure-starters/spring-cloud-starter-azure-appconfiguration-config/README.md

```diff
@@ -60,11 +60,11 @@ spring.cloud.azure.appconfiguration.managed-identity.client-id | Client id of th
 
 Name | Description | Required | Default
 ---|---|---|---
-spring.cloud.azure.appconfiguration.stores[0].endpoint | Endpoint of the configuration store, required when `connection-string` is empty. If `connection-string` is empty and application is deployed on Azure VM or App Service with managed identity enabled, will try to load `connection-string` from Azure Resource Management. | Conditional | null
+spring.cloud.azure.appconfiguration.stores[0].endpoint | Endpoint of the configuration store, required when `connection-string` is empty. If `connection-string` is empty alternate connection methods will be attempted based on configuration, such as System Assigned Identity, or User Assigned Identity if the client-id value is set. | Conditional | null
```

Comment:
I think we should reword the description and focus on explaining when the endpoint should be used, for example,

*When the endpoint of an App Configuration store is specified, a managed identity or a token credential provided using `AppConfigCredentialProvider` will be used to connect to the App Configuration service. Errors (what error?) will be thrown if the endpoint and connection-string are specified at the same time.*

We don't have to explain the differences between system-assigned managed identity and user-assigned managed identity here.